connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -203,6 +204,19 @@ message Sort {
   }
 }
 
+
+// Drop specified columns.
+message Drop {
+  // (Required) The input relation.
+  Relation input = 1;
+
+  // (Required) columns to drop.
+  //
+  // Should contain at least 1 item.
+  repeated Expression cols = 2;
```

Comment:
Wondering if the name should be more explicit like "dropped"?

Reply:
here follows the naming in `def drop(col: Column, cols: Column*): DataFrame`